src/components/ReportTransaction.js

```diff
@@ -63,10 +85,24 @@ class ReportTransaction extends Component {
                                 styles.buttonSmall,
                                 styles.chatItemComposeSecondaryRowOffset,
                                 styles.mb3,
+                                styles.rejectIOU,
                             ]}
                             onPress={this.rejectTransaction}
                         >
-                            <Text style={[styles.buttonSmallText]}>{this.props.rejectButtonLabelText}</Text>
+                            {
+                            this.isRejected(this.props.action.originalMessage.IOUTransactionID, rejectedIDs)
```

Comment:
Above I asked if you can move `this.props.action.originalMessage.IOUTransactionID` into the `this.isRejected()` function. Can you also use `lodashGet` to get this prop's data to set a default & avoid crashes if any prop in the chain is undefined?

Reply:
Thanks for the feedback! 
Yes, I'll do that.